MCP4725.cpp

```diff
@@ -70,6 +70,27 @@ bool MCP4725::begin(const uint8_t dataPin, const uint8_t clockPin)
 }
 #endif
 
+#if defined (ARDUINO_ARCH_RP2040)
+
+bool MCP4725::begin(int sda, int scl)
```

Comment:
Please add a parameter for useWire1 in

begin(int sda, int scl, int wireUsed)
{
  if wireUsed == 1 ==> _wire = Wire1
  else _wire = Wire0
Otherwise one has to call selectWire1() before begin() and that is not what people are used to. The two select functions are not needed then.
Would be a simpler API imho.

(identical to other PR)

Reply:
solved